modules/widgets/contact-info.php

```diff
@@ -59,6 +59,7 @@ public function defaults() {
 				'address' => __( "3999 Mission Boulevard,\nSan Diego CA 92109", 'jetpack' ),
 				'phone'   => _x( '1-202-555-1212', 'Example of a phone number', 'jetpack' ),
 				'hours'   => __( "Lunch: 11am - 2pm \nDinner: M-Th 5pm - 11pm, Fri-Sat:5pm - 1am", 'jetpack' ),
+				'email'   => get_bloginfo( 'admin_email' ),
```

Comment:
I'm not crazy about using the admin email as the default.  Why not just do something like `jane.doe@gmail.com`?